com.github.pyfa_org.Pyfa.json

```diff
@@ -0,0 +1,347 @@
+                  "cflags-override": true,
+                  "cxxflags": "-O2 -g -pipe -DENABLE_YARR_JIT=0 -Wno-expansion-to-defined -Wno-class-memaccess -fno-delete-null-pointer-checks -fpermissive -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_60",
+                  "cxxflags-override": true,
+                  "config-opts": [
+                      "--disable-jit"
+                  ]
+              }
+          }
+      },
+      "cleanup": [
+          "/share/doc",
+          "/bin"
+      ],
+      "sources": [
+          {
+              "type": "archive",
+              "url": "https://www.webkitgtk.org/releases/webkitgtk-2.4.11.tar.xz",
+              "sha256": "588aea051bfbacced27fdfe0335a957dca839ebe36aa548df39c7bbafdb65bf7"
+          },
+          {
+              "type": "patch",
+              "path": "patches/webkitgtk-2.4.11.patch"
+          },
+          {
+              "type": "patch",
+              "path": "patches/webkitgtk-fix.patch"
+          },
+          {
+              "type": "patch",
+              "path": "patches/webkitgtk-41-fix.patch"
+          }
+      ]
+    },
+    "python3-modules.json",
+    {
+      "name": "pyfa-2.53.0",
+      "sources" : [
+        {
+          "url" : "https://github.com/pyfa-org/Pyfa/archive/refs/tags/v2.53.0.tar.gz",
+          "sha256" : "82958260289cde0ee0894f658a4173d57e6e343c8505b5fbcbbaa0f7a917f96d",
+          "type" : "archive"
+        },
+        {
+          "type": "file",
+          "path": "pyfa"
+        },
+        {
+          "type": "file",
+          "path": "com.github.pyfa_org.Pyfa.desktop"
+        },
+        {
+            "type": "file",
+            "path": "com.github.pyfa_org.Pyfa.metainfo.xml"
+        },
+        {
+          "type": "file",
+          "path": "icons/com.github.pyfa_org.Pyfa-256.png"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "find locale/ -type f -name '*.po' -exec msgen '{}' -o '{}' \\;",
+        "/app/bin/python3.7 scripts/compile_lang.py",
+        "/app/bin/python3.7 db_update.py",
+        "mkdir -p ${FLATPAK_DEST}/lib/pyfa",
+        "cp -r eos graphs gui imgs locale service utils eve.db config.py pyfa.py db_update.py README.md LICENSE version.yml ${FLATPAK_DEST}/lib/pyfa",
+        "find ${FLATPAK_DEST}/lib/pyfa | grep -E '(__pycache__|\\.pyc|\\.pyo$)' | xargs rm -rf",
+        "install -D --mode=0755 pyfa /app/bin/pyfa",
+        "install -D -t \"${FLATPAK_DEST}/share/applications/\" ./com.github.pyfa_org.Pyfa.desktop",
+        "install -D ./com.github.pyfa_org.Pyfa-256.png \"${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/com.github.pyfa_org.Pyfa.png\"",
+        "install -D -t \"${FLATPAK_DEST}/share/metainfo/\" ./com.github.pyfa_org.Pyfa.metainfo.xml"
+      ]
+    }
+  ],
+  "cleanup" : [
```

Comment:
cleanup should be above modules.

Also it contain a long list of things that don't even seem to be built so maybe it should be trimmed to what's actually needed.

Reply:
Maybe this list looks excessive, but i tried to reduce final size of application. All these files are present in build and most of them are installed with python deps.

Reply:
Cleanup array moved above modules